site/en/docs/extensions/reference/web_suplement/index.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: 'layouts/doc-post.njk'
+title: Web platform suplement
```

Comment:
When we discussed this in the last writing meeting, our conclusion was to move forward with documenting web platform features as how-to guides (which I think your `extensions/mv3/file_handling` here does well) and to come back to if we want a web supplement section in the future. With that in mind, would you be comfortable removing those two files from this PR? I want to avoid committing to anything there, especially while Sebastian is out.

Reply:
I thought I had removed this. Sorry.